src/main/kotlin/com/team4099/robot2025/subsystems/arm/Arm.kt

```diff
@@ -0,0 +1,88 @@
+package com.team4099.robot2025.subsystems.arm
+
+import com.team4099.robot2025.subsystems.superstructure.Request
+import com.team4099.robot2025.util.CustomLogger
+import org.littletonrobotics.junction.Logger
+import org.team4099.lib.units.derived.Angle
+import org.team4099.lib.units.derived.ElectricalPotential
+import org.team4099.lib.units.derived.degrees
+import org.team4099.lib.units.derived.volts
+
+class Arm(val io: ArmIO) {
+    val inputs = ArmIO.ArmIOInputs()
+
+
+    var armTargetVoltage: ElectricalPotential = 0.0.volts
+    var armTargetPosition: Angle = 0.0.degrees
+    var armTolerance: Angle = 0.0.degrees
+
+    var currentState = ArmState.UNINITIALIZED
+    var currentRequest: Request.ArmRequest = Request.ArmRequest.Zero()
+        set(value) {
+            when (value) {
+                is Request.ArmRequest.OpenLoop -> {
+                    armTargetVoltage = value.armVoltage
+                }
+
+                is Request.ArmRequest.CloseLoop -> {
+                    armTargetPosition = value.armPosition
+                    armTolerance = value.armTolerance
```

Comment:
what is this for.

Reply:
yeah i dont think you need this esp with motionmagic targeting a specific position consistently. this a remnant of p5